Bottom: 5786ad886bcc639ffdb0eddddaa7aa98ebf2df9c
Top:    5485af5a69865b734c708f14b1fa410109cff953
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-12 01:15:47 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
